### DISCUSSION TITLE
文章内页面与主页及设置不符

### DISCUSSION THREAD

**Discussion by lopkuiughio** [YunYouJun/hexo-theme-starry#365] [Q&A 疑难杂症]
作者大大，我在创建了一篇文章后，部署完进入页面发现没有开的mickey_mouse出现在了侧边栏，跟主页侧边栏不一样
还有主页流动的云颜色怎么改啊，无论是写单词还是十六进制都没有用啊

**Comment by YunYouJun** [comment]
yaml 中配置十六进制颜色需要加引号。

**Reply by YunYouJun** [reply]
出现的问题最好可以截图示意一下。
